src/main/java/core/basesyntax/service/impl/WriteDataImpl.java

```diff
@@ -13,7 +13,7 @@ public boolean writeData(List<String> list, String fileName) {
             Files.write(Path.of(fileName), list);
             return true;
         } catch (IOException e) {
-            return false;
+            throw new RuntimeException("It is not possible to write this file " + fileName);
```

Comment:
```suggestion
            throw new RuntimeException("It is not possible to write this file " + fileName, e);
```
завжди потрібно прокидати початкову помилку